convert2rhel/utils.py

```diff
@@ -496,21 +496,35 @@ def download_pkg(
     if ret_code != 0:
         loggerinst.warning("Output from the yumdownloader call:\n%s" % (output))
 
-        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:
+        # add in the new envar that should be changed to along with the old one
+        allow_old_envar_names = ("CONVERT2RHEL_INCOMPLETE_ROLLBACK", "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK")
+
+        # search for either envar in the system to see if it has been set
+        if any(envvar in os.environ for envvar in allow_old_envar_names):
+
+            # check to see which envar was used
+            if "CONVERT2RHEL_INCOMPLETE_ROLLBACK" not in os.environ:
+                # give the warning about the old envar not being used anymore if it is uesed
+                loggerinst.warning(
+                    "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'"
+                    " environment variable.  Please switch to 'CONVERT2RHEL_INCOMPLETE_ROLLBACK'"
+                    " instead."
+                )
+
+                # give regular warning if the new envar is used
+                loggerinst.warning(
+                    "Couldn't download the %s package. This means we will not be able to do a"
+                    " complete rollback and may put the system in a broken state.\n"
+                    "'CONVERT2RHEL_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion." % (pkg)
+                )
```

Comment:
Take a look at your indentation in this section from line 503 to 519 and compare it to what you want to have happen.  What you want is:
* If either the deprecated or new environment variable is set, then you will log the warning that is on line 515.
* If the deprecated environment variable is set, then you will also warn that the environment variable is deprecated and they should use the new environment variable instead.

The indentation of your statements doesn't quite match with that.  When you change your unit test to check for the proper log message with each of the environment variables, you'll find that the new environment variable isn't logging the proper warning.

Reply:
yeah I think I noticed this in the other PR for the log message in checks.py! Thank you I will go back and get this fixed!